nsqadmin/http.go

```diff
@@ -1139,6 +1184,51 @@ func (s *httpServer) statisticsHandler(w http.ResponseWriter, req *http.Request,
 		messages = append(messages, pe.Error())
 	}
 
+	if filter == "channel-timeout" || filter == "channel-requeue" {
```

Comment:
Why not move sort code to the `GetNSQDStats` just like others and only handle removing the 0 time/req here.